apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -1,6 +1,7 @@
 import { zodResolver } from "@hookform/resolvers/zod";
 import { EventTypeCustomInputType, WorkflowActions } from "@prisma/client";
 import { SchedulingType } from "@prisma/client";
+import { useMutation } from "@tanstack/react-query";
```

Comment:
Is there a reason this import is now coming from here? @KATT

Reply:
https://tanstack.com/query/v4/docs/guides/migrating-to-react-query-4